server/index.js

```diff
@@ -3,6 +3,7 @@ const app = express();
 const morgan = require('morgan');
 const parser = require('body-parser');
 const db = require('../database/index.js');
+
 let port = 3031;
 
 if (port === null || port === '') {
```

Comment:
I'm confused by this code. Why is this here if you're setting your port above?

Reply:
Changed.